systemvm/patches/debian/config/opt/cloud/bin/cs/CsRedundant.py

```diff
@@ -96,7 +96,8 @@ def _redundant_on(self):
                 d = s.replace(".templ", "")
             CsHelper.copy_if_needed(
                 "%s/%s" % (self.CS_TEMPLATES_DIR, s), "%s/%s" % (self.CS_ROUTER_DIR, d))
-        CsHelper.copy(
```

Comment:
What fixed the RVR was the change in the CsRedundant._collect_ips function. That one was kept, I just fixed the rVPC part of it, which was changed with no reason.

Concerning the template file (conntrackd.conf.templ), that's just a default conntrackd file completely commented out. If we copy it in every operation in the routers it will restart conntrackd every time, because it copies the the and afterwards it applies the configuration we need:

```
        # conntrackd configuration
        connt = CsFile(self.CONNTRACKD_CONF)
        if guest is not None:
            connt.section("Multicast {", "}", [
                          "IPv4_address 225.0.0.50\n",
                          "Group 3780\n",
                          "IPv4_interface %s\n" % guest.get_ip(),
                          "Interface %s\n" % guest.get_device(),
                          "SndSocketBuffer 1249280\n",
                          "RcvSocketBuffer 1249280\n",
                          "Checksum on\n"])
            connt.section("Address Ignore {", "}", self._collect_ignore_ips())
            connt.commit()
```

The copy if needed is fine.